.changes/unreleased/Under the Hood-20220606-230353.yaml

```diff
@@ -1,5 +1,5 @@
-kind: Under the Hood
-body: Update context readme + clean up context code"
+kind: Security
```

Comment:
nit: This won't change how the changelog markdown gets generated but for consistency can we change the file name to be `Security-20220606-230353.yaml`?  For context, the filenames are just like this for uniqueness, they don't actually get any fields out of the name.